packages/docs/docs/200-security/registered-queries.md

```
@@ -0,0 +1,149 @@
+```ts
+const findUserById = (id: number) => from('users').filter({id}).first()
+
+// The QueryEngine will substitute the parameter values into the query and execute it.
+queryEngine.registerQuery(findUserById(0))
+
+
+```
+
+### What happens in case of a hash collision?
+
+If you try to register two different queries with the same hash, the QueryEngine will throw an error. 
```

Markdownlint

Expected: 0 or 2; Actual: 1
Trailing spaces

(MD009, no-trailing-spaces)